### ISSUE TITLE
Registry -- allow users to query beta mines

### ISSUE THREAD

**Issue by julie-sullivan** [intermine/InterMineR#46]
Currently you can only query production InterMines. There is a use case for being able to query dev mines, e.g FlyMine beta.

**Comment by yochannah** [comment]
I've added the hacktoberfest label to this issue - if you're interested in contributing, the code to fetch a list of InterMines from the InterMine registry is here: https://github.com/intermine/InterMineR/blob/master/R/InterMineR.R

Context: when using methods that access the registry, it would be nice to have an optional argument that allows users to select all InterMines, _just_ the dev intermines, or _just_ the production InterMines.

Registry docs are here - http://registry.intermine.org/api-docs/ - but probably all you'll need is to switch the param at the end of the URL

- http://registry.intermine.org/service/instances?mines=dev
- http://registry.intermine.org/service/instances?mines=prod
- http://registry.intermine.org/service/instances?mines=all


## if you want to pick up this task

0. Take a look at the [InterMine contributing guidelines](http://intermine.org/contributing/)
1. Comment on this issue stating that you intend to work on the task
2. When you're ready, add your work to the repo and create a pull request.

### What to do if you need help

Mention @yochannah, tweet [@yoyehudi](http://www.twitter.com/yoyehudi), pop by to say hi on [chat](http://chat.intermine.org) or if needed email yo@intermine.org. Don't forget we're usually only available during uk office hours and will not be able to respond at other times :)

**Comment by lvclark** [comment]
I would be interested in taking this on for Hacktoberfest if it would still be helpful!

**Comment by rachellyne** [comment]
Yes, this would still be useful.  Thank you.   Yo Yehudi has now left the InterMine project so please use the following contact details for any help:  dev@lists.intermine.org or the above chat link or comment here with @rachellyne or @danielabutano or @uosl
